crates/pallet-executor/src/lib.rs

```diff
@@ -203,6 +238,20 @@ mod pallet {
     #[pallet::getter(fn executor)]
     pub(super) type Executor<T: Config> = StorageValue<_, (T::AccountId, ExecutorId), OptionQuery>;
 
+    /// Mapping from the primary block number to the corresponding verified execution receipt.
+    #[pallet::storage]
+    pub(super) type Receipts<T: Config> =
+        StorageMap<_, Twox64Concat, BlockNumber, ExecutionReceipt<T::SecondaryHash>, OptionQuery>;
+
+    /// Number of execution receipts kept in the state.
```

Comment:
Shouldn't this be just a constant though?

Reply:
This number of receipts stored in the state is kind of the challenge period of the receipts. Storing it as a storage item gives you more flexibility to tweak it accordingly since a sudo interface is provided.

Reply:
Right, but we can also tweak it with runtime upgrade and I don't expect us to tinker with it outside of runtime upgrade anyway.

Reply:
Hmm, I think it's very likely we'll tweak this parameter multiple times to find a proper value in a test environment without having to perform a runtime upgrade, so it's helpful before we find a sensible parameter IMHO. But I don't insist on this way as you can do it via runtime upgrade for sure.